README.md

```diff
@@ -188,16 +188,45 @@ root@vm2:~$
 	ssh root@IP_of_my_VM
 	```
 
-	You will be asked to enter the password that you set up the first time you logged in as root (see the section "Using a VM for the First Time" above). 
+	You will be asked to enter the password that you set up the first time you logged in as root (see the section "Using a VM for the First Time" above).
 
 1. To start installing software, enter the following lines in the terminal:
 	```bash
 	cd
 	mkdir git
 	cd git
-	apt get install git
-	git clone https://github.com/floswald/cumulus-setup.git 
+	apt-get install git
+	git clone https://github.com/floswald/cumulus-setup.git
 	cd cumulus-setup
 	./install.sh
 	```
 	You will be asked to enter `Y` (yes) several times. When the installation is done, you can check that everything went fine by simply entering `python`, or `R` in the terminal.
+
+## Julien's experience
```

Comment:
this is useful . the place for this is the wiki though. can you move this there? just create a new page "julien's experience" or so. thanks!